src/components/explorer/ExplorerDatasetSearch.js

```diff
@@ -99,6 +73,39 @@ class ExplorerDatasetSearch extends Component {
     }
 
     render() {
+
+        const {sortColumnKey, sortOrder} = this.props.tableSortOrder;
+
+        const SEARCH_RESULT_COLUMNS = [
```

Comment:
change this to camel case since it's no longer global const for the file. other than that looks good!